Sources/MapboxDirections/CoordinateBounds.swift

```diff
@@ -59,8 +59,14 @@ extension CoordinateBounds: Codable {
     
     public init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: CodingKeys.self)
-        southWest = try container.decode(CLLocationCoordinate2D.self, forKey: .southWest)
-        northEast = try container.decode(CLLocationCoordinate2D.self, forKey: .northEast)
+        southWest = try container.decode(CLLocationCoordinate2DCodable.self, forKey: .southWest).decodedCoordinates
+        northEast = try container.decode(CLLocationCoordinate2DCodable.self, forKey: .northEast).decodedCoordinates
```

Comment:
It would be tempting to remove `CoordinateBounds` in favor of the `BoundingBox` struct in Turf. Unfortunately, it looks like `BoundingBox` was copied from `CoordinateBounds` back before we fixed #348, so we’ll have to fix mapbox/turf-swift#102 before we can rely on `BoundingBox`.